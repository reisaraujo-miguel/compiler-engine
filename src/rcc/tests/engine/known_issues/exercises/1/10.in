80
2
s
m
0
