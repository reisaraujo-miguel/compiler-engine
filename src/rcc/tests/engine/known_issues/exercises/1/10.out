Peso (kg): Número de copos de 150ml de cerveja consumidos: Em jejum (S/N)? Sexo (M/F)? 
Taxa de alcoolemia: 0.2 g/l
Pode dirigir. Você é uma pessoa responsável.

Peso (kg):
